modules/eco-node-health-check-operator-about.adoc

```diff
@@ -58,7 +58,40 @@ During the upgrade process, nodes in the cluster might become temporarily unavai
 [id="understanding-nhc-operator-workflow_{context}"]
 == Understanding the Node Health Check Operator workflow
 
-When a node is identified as unhealthy, the Operator checks how many other nodes are unhealthy. If the number of healthy nodes exceeds the amount that is specified in the `minHealthy` field of the `NodeHealthCheck` CR, the controller creates a remediation CR from the details that are provided in the external remediation template by the remediation provider. After remediation, the node's health status is updated accordingly.
+When a node is identified as unhealthy, the Node Health Check Operator checks how many other nodes are unhealthy. If the number of healthy nodes exceeds the amount that is specified in the `minHealthy` field of the `NodeHealthCheck` CR, the controller creates a remediation CR from the details that are provided in the external remediation template by the remediation provider. After remediation, the kubelet updates the node's health status.
 
-When the node turns healthy, the controller deletes the external remediation template
-and updates the node's health status.
\ No newline at end of file
+When the node turns healthy, the controller deletes the external remediation template.
+
+[id="preventing-nhc-conflict-with-mhc_{context}"]
+== Preventing node health check conflicts with machine health check
+
+When both, node health check and machine health check are deployed, the node health check avoids conflict with the machine health check.
+
+[NOTE]
+====
+{product-title}, deploys `machine-api-termination-handler` as the default `MachineHealthCheck` resource.
+====
+
+The following list summarizes the system behavior when node health checks and machine health checks are deployed:
+
+* If only the default machine health check exists, the node health check continues to identify unhealthy nodes. However, the node health check ignores unhealthy nodes in a `Terminating` condition. The default machine health check handles the unhealthy nodes with a `Terminating` condition.
++
+.Example log message
+[source,terminal]
+----
+INFO MHCChecker	ignoring unhealthy Node, it is terminating and will be handled by MHC	{"NodeName": "node-1.example.com"}
+----
+
+* If the default machine health check is modified (for example, the `unhealthyConditions` is  `Ready`), or if additional machine health checks are created, the node health check is disabled.
++
+.Example log message
+----
+INFO controllers.NodeHealthCheck disabling NHC in order to avoid conflict with custom MHCs configured in the cluster {"NodeHealthCheck": "/nhc-worker-default"}
+----
+
+* When, again, only the default machine health check exists, the node health check is re-enabled.
```

Comment:
I'm not sure I understand what this means. The user removes the modifications to the default machine health check, or deletes the modified default health check, something else??

Reply:
all 3: user remove modifications, or deletes default MHC, or deletes addtional MHCs. It all boils down to: when no or only unmodified default MHC exists, NHC works. Otherwise not.